docs/blocks/blocks-experimental.md

```diff
@@ -78,541 +171,547 @@ Adds a mob effect to a target when triggered.
+    }
   }
 }
 ```
 
-For block rotations like logs, see [this](/blocks/rotatable-blocks)
-
-### minecraft:unit_cube
-
-Forces the block to be a cube with face culling.
-
-<CodeHeader></CodeHeader>
+Or...
 
 ```json
 {
-  "minecraft:unit_cube": {}
+  "wiki:trigger_event": {
+    "trigger": "wiki:my_event" // Trigger another block event
+  }
 }
 ```
 
-### minecraft:on_step_on
+### minecraft:on_fall_on
 
-Runs an event when an entity stepped on the block.
+:::warning EXPERIMENTAL
+This component requires the `Holiday Creator Features` experiment to be enabled.
+:::
+
+Runs an event when an entity fell on the block.
 
 <CodeHeader></CodeHeader>
 
 ```json
 {
-  "minecraft:on_step_on": {
-    "event": "block_event",
-    "target": "self",
-    "condition": "q.block_property('wiki:block_property')" //Optional
+  "minecraft:on_fall_on": {
+    "event": "wiki:example_event",
+    "target": "self", // Optional - 'self' is default (targets block)
+    "condition": "query.block_property('wiki:boolean_property_example')", // Optional
+    "min_fall_distance": 5
   }
 }
 ```
 
-Note: you need entity collision not to be set false to trigger event.
+### minecraft:on_interact
 
-### minecraft:on_step_off
+:::warning EXPERIMENTAL
+This component requires the `Holiday Creator Features` experiment to be enabled.
+:::
 
-Runs an event when an entity stepped off of the block.
+Runs an event when he Player uses the block.
 
 <CodeHeader></CodeHeader>
 
 ```json
 {
-  "minecraft:on_step_off": {
-    "event": "block_event",
-    "target": "self",
-    "condition": "q.block_property('wiki:block_property')" //Optional
+  "minecraft:on_interact": {
+    "event": "wiki:example_event",
+    "target": "self", // Optional - 'self' is default (targets block)
+    "condition": "query.block_property('wiki:boolean_property_example')" // Optional
   }
 }
 ```
 
-Note: you need entity collision not to be set false to trigger event.
+### minecraft:on_placed
 
-### minecraft:on_fall_on
+:::warning EXPERIMENTAL
+This component requires the `Holiday Creator Features` experiment to be enabled.
+:::
 
-Runs an event when an entity fell on the block.
+Runs an event when an entity placed the block.
 
 <CodeHeader></CodeHeader>
 
 ```json
 {
-  "minecraft:on_fall_on": {
-    "event": "block_event",
-    "target": "self",
-    "condition": "q.block_property('wiki:block_property')", //Optional
-    "min_fall_distance": 5
+  "minecraft:on_placed": {
+    "event": "wiki:example_event",
+    "target": "self", // Optional - 'self' is default (targets block)
+    "condition": "query.block_property('wiki:boolean_property_example')" // Optional
   }
 }
 ```
 
-### minecraft:on_placed
+### minecraft:on_player_destroyed
 
-Runs an event when an entity placed the block.
+:::warning EXPERIMENTAL
+This component requires the `Holiday Creator Features` experiment to be enabled.
+:::
+
+Runs an event when the player destroys the block.
```

Comment:
In line 538, Player was capitalized. Either make line 538 lowercase or make this Capitalized